### DISCUSSION TITLE
anyway to use with supabase storage cdn ?

### DISCUSSION THREAD

**Discussion by thititongumpun** [ascorbic/unpic-img#698] [Q&A] [answered]
i tried to use supabase cdn but its generate <img /> src is wrong from url storage so its cannot use

**Comment by ascorbic** [comment]
Can you give more details? What's the source URL, what does it generate, what do you expect it to generate?

**Reply by thititongumpun** [reply]
source URL: <https://fldpwjgfpmwbtlptuefj.supabase.co/storage/v1/object/public/memes_bucket/images/teeyodnadeh1>
![image](https://github.com/user-attachments/assets/86219440-ee3a-43f8-951c-3325c53ab458)

but image generate: <https://fldpwjgfpmwbtlptuefj.supabase.co/storage/v1/render/image/public/memes_bucket/images/teeyodnadeh12?width=640&height=480>
<img width="651" alt="image" src="https://github.com/user-attachments/assets/d2576a3b-2928-4f05-8a57-f35bc86b211b">

**Reply by ascorbic** [reply, marked as answer]
That URL is correct: when you pass a supabase URL it will use Supabase Storage Image Transformations. The problem is that you haven't got that feature enabled on your account. What are you wanting it to use for transforms? https://supabase.com/docs/guides/storage/serving/image-transformations

**Reply by thititongumpun** [reply]
but i used this url: <https://fldpwjgfpmwbtlptuefj.supabase.co/storage/v1/object/public/memes_bucket/images/teeyodnadeh1> in normal img tag it works without using Supabase Image Transformations
so can i still use url: <https://fldpwjgfpmwbtlptuefj.supabase.co/storage/v1/object/public/memes_bucket/images/teeyodnadeh1> in 
import { Image } from "@unpic/preact"; 
without compile Image component to img tag 
src =<https://fldpwjgfpmwbtlptuefj.supabase.co/storage/v1/render/image/public/memes_bucket/images/teeyodnadeh12?width=640&height=480>

**Reply by ascorbic** [reply]
The point of Unpic is that it detects the host of an image and generates the URL to display the sizes needed. In this case it's detecting supabase, and generating the correct URLs to resize them to the required size.

**Reply by thititongumpun** [reply]
I see thanks.
